loom/script/runtime/lsLuaState.cpp

```diff
@@ -96,9 +97,11 @@ void LSLuaState::open()
     luaopen_math(L);
     luaL_openlibs(L);
 
+#ifdef LUAJIT_MODE_MASK
```

Comment:
To be honest, I only added that ifdef because it was breaking the JIT-less build ;)